api/middlewares/cookieHandler.js

```diff
@@ -0,0 +1,6 @@
+const cookieHandler = (req, res, next) => {
+    req.parsedCookies = req.cookies;
```

Comment:
I guess your task here was to get the cookies from the request header and parse it yourself.